src/styles/main.scss

```diff
@@ -0,0 +1,124 @@
+body {
+  margin: 0;
+  width: 1024px;
+  height: 100%;
+  animation: pulse 5s infinite;
+}
+  
+@keyframes pulse {
+  0% {
+    background-color: #001f3f;
+  }
+  100% {
+    background-color: #ff4136;
+  }
+}
+
+.city {
+  display: flex;
+}
+
+h1 {
```

Comment:
Replace the 'h1' selector with a class selector following the BEM methodology, e.g. '.stopwatch__title'.